confgen/sample-proxyfs-configuration/proxyfs-shares.conf

```diff
@@ -7,12 +7,12 @@ ReadCacheLineSize: 1048576
 ReadCacheWeight:   100
 PrimaryPeer: c67631b5-cb88-11e9-99da-0248604d6797
 VolumeList: vol-vg32-2
-VirtualIPAddr:  32.32.32.32
+VirtualIPAddr:  32.32.32.32/24
```

Comment:
Should we perhaps not name this "VirtualIPAddr" given that it also includes the netmask (in this case /24)?

Reply:
We could call it `VirtualIPAddrInCIDRNotation` or, perhaps, `VirtualCIDRIPAddr`.

But that requries changes to the controller, monitord, and confgen and is really outside the scope of this work.